fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/values/NumericAggregationValue.java

```diff
@@ -722,7 +834,11 @@ public NumericAccumulator(@Nonnull final PhysicalOperator physicalOperator) {
 
         @Override
         public void accumulate(@Nullable final Object currentObject) {
-            this.state = physicalOperator.evalPartialToPartial(state, currentObject);
+            if (state == null) {
+                this.state = physicalOperator.evalInitialToPartial(currentObject);
+            } else {
+                this.state = physicalOperator.evalPartialToPartial(state, currentObject);
+            }
```

Comment:
modify this function, because the old one seems to neglect the `evalInitialToPartial` scenario.